IdentityCore/src/requests/broker/MSIDSSOExtensionSignoutRequest.m

```diff
@@ -89,6 +97,13 @@ - (nullable instancetype)initWithRequestParameters:(nonnull MSIDInteractiveReque
             }
             
             __typeof__(self) strongSelf = weakSelf;
+            
+#if !EXCLUDE_FROM_MSALCPP
+            [operationResponse trackPerfTelemetryWithLastRequest:strongSelf.lastRequestTelemetry
+                                                requestStartDate:strongSelf.requestSentDate
+                                                            type:MSID_PERF_TELEMETRY_SIGNOUT_TYPE];
+#endif
+            
```

Comment:
Looks like this code : 

```
[operationResponse trackPerfTelemetryWithLastRequest:strongSelf.lastRequestTelemetry
                                                requestStartDate:strongSelf.requestSentDate
                                                            type:MSID_PERF_TELEMETRY_SIGNOUT_TYPE];

_lastRequestTelemetry = [MSIDLastRequestTelemetry sharedInstance];

self.requestSentDate = [NSDate date];
```
is repeated across Request classes. Can it be shifted to a common utility class with the request object as a parameter?

Reply:
I already shifted the maximum code that I could to the operationResponse class as a utility. The rest cannot be shifted without a bigger refactoring of those classes, which I don't think is worth doing here.